package-lock.json

```diff
@@ -1,11226 +1,14 @@
 {
   "name": "response-operations-ui",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You sure this is right? How did you generate this? Having the lockfileversion go to 1 seems wrong, but I don't know enough to say one way or the other

Reply:
This has been reverted back. There was an issue with my local npm that caused this to be changed.